CopilotKit/packages/react-textarea/src/hooks/make-autosuggestions-function/use-make-standard-insertion-function.tsx

```diff
@@ -67,8 +67,14 @@ export function useMakeStandardInsertionOrEditingFunction(
           },
         ];
 
-        const apiEndpoint = ChatlikeApiEndpoint.fromCopilotApiConfig(copilotApiConfig);
```

Comment:
same as above - we're now ignoring some `copilotApiConfig` properties

Also, whatever we implement for the insertion function, we also need to implement for the editing function below (`const editingFunction = useCallback...`)